assets/js/components/utils/index.js

```diff
@@ -96,3 +96,14 @@ export const getProductTags = ( { selected = [], search } ) => {
 		return uniqBy( flatten( data ), 'id' );
 	} );
 };
+
+/**
+ * Get a promise that resolves to a category object from the API.
+ *
+ * @param {Object} categoryId Id of the product to retrieve.
```

Comment:
`categoryId` will be a string or a number instead of an object, right?

I saw that's an issue in `getProduct`'s description too, so if you could fix it there too :+1: .